matter_server/server/device_controller.py

```diff
@@ -1244,8 +1244,6 @@ async def _node_offline(self, node_id: int) -> None:
             await self._call_sdk(sub.Shutdown)
         # mark node as unavailable
         node = self._nodes[node_id]
-        if not node.available:
-            return  # nothing to do to
```

Comment:
Since available is the only change we do this node, I still feel we should only signal an update if this is indeed changing?

I guess we don't expect for `_node_offline` to get called twice typically (anymore), but it also shouldn't hurt? :thinking: